src/frequenz/dispatch/actor.py

```diff
@@ -65,6 +65,7 @@ def __init__(
 
         self._client = client
         self._dispatches: dict[int, Dispatch] = {}
+        self._dispatch_type_ids_map: dict[str, list[int]] = {}
```

Comment:
I would add documentation about what this is for, as it is not trivial to figure it out.